spk/tvheadend/src/service-setup.sh

```diff
@@ -42,6 +47,18 @@ service_postinst ()
     $BIN/busybox --install $BIN
     $BIN/delgroup "${USER}" "users" >> ${INST_LOG}
     $BIN/deluser "${USER}" >> ${INST_LOG}
```

Comment:
this is old dsm5->dsam6 migration stuff and can be removed (we do not support dsm5->dsm6 migration anymore).

and dependency to busybox can be removed

Reply:
Changes already present in #4545